src/licensedcode/detection.py

```diff
@@ -295,19 +673,154 @@ def is_license_intro(license_match):
     )
 
 
-class CombinationReason(Enum):
-    NOT_COMBINED = 'not-combined'
-    UNKNOWN_INTRO_FOLLOWED_BY_MATCH = 'unknown-intro-followed-by-match'
-    CONTAINED_SAME_LICENSE = 'contained-with-same-license'
-    NOTICE_FOLLOWED_BY_TEXT = 'notice-followed-by-text'
-    CONTIGUOUS_SAME_LICENSE = 'contiguous-with-same-license'
-    REF_FOLLOWED_BY_NOTICE = 'ref-followed-by-notice'
-    REF_FOLLOWED_BY_TEXT = 'ref-followed-by-text'
-    TAG_FOLLOWED_BY_NOTICE = 'tag-followed-by-notice'
-    TAG_FOLLOWED_BY_TEXT = 'tag-followed-by-text'
-    TAG_FOLLOWED_BY_REF = 'tag-followed-by-ref'
-    UNKNOWN_FOLLOWED_BY_MATCH = 'unknown-ref-followed-by-match'
-    UNVERSIONED_FOLLOWED_BY_VERSIONED = 'un-versioned-followed-by-versioned'
+def is_license_reference_local_file(license_match):
+    """
+    Return True if `license_match` LicenseMatch dict has a
+    non-empty `referenced_filename`, i.e. contains a license
+    reference to a local file.
+    """
+    if license_match['matched_rule']['referenced_filenames']:
+        return True
+
+    return False
+
+
+def combine_license_references(license_matches):
```

Comment:
This is not combining, but only filtering

Reply:
Changing this to filtering.